Dockerfile

```diff
@@ -0,0 +1,15 @@
+# Pull base image.
+FROM ubuntu:16.04
+# Install.
+RUN \
+  apt-get update && \
+  apt-get -y upgrade && \
+  apt-get install -y curl
+  
+# Add files.
+WORKDIR /home
+RUN curl -sSL https://get.haskellstack.org/ | sh
```

Comment:
what's the purpose of downloading stack here? i suppose i haven't tried running the binary on a machine without it, but i didn't expect it to be needed

Reply:
Hello, ok i will update the readme too.
And for the stack, indeed it is not necessary anymore. The reason it was there is that in my first dockerfile I compliled it when building the docker image. But it took to long imho so I switched to just coping the executable.
I will remove that line too.